src/mpid/ch4/shm/ipc/src/globals.c

```diff
@@ -3,9 +3,9 @@
  *     See COPYRIGHT in top-level directory
  */
 
-#ifndef XPMEM_IMPL_H_INCLUDED
-#define XPMEM_IMPL_H_INCLUDED
```

Comment:
Why are these header guards deleted instead of being renamed?

Reply:
`xpmem_impl.h` is empty and has been deleted. It might be a git bug to link it with `ipc/src/globals.c `. In `ipc/src/globals.c `, no guards is needed.

Reply:
This should be a mistake

Reply:
@gcongiu It is not a mistake by us. The `xpmem_impl.h` has been deleted. It is correct that the guards should not be added for `globals.c`